universal-application-tool-0.0.1/app/views/components/Icons.java

```diff
@@ -11,6 +11,10 @@
   public static final String CHECKBOX_SVG_PATH = "M5 13l4 4L19 7";
   // Menu
   public static final String DROPDOWN_SVG_PATH = "M4 6h16M4 10h16M4 14h16M4 18h16";
+  public static final String FILEUPLOAD_SVG_PATH =
```

Comment:
suggestion (non-blocking): Could you write the name of the icon you chose here?

Reply:
Done. I didn't notice the name is important but I can see it's nice to reference in code.